Notepad/HomeViewController.swift

```diff
@@ -17,20 +17,32 @@ class HomeViewController: UIViewController {
     }
     
     @IBAction func handleSaveButtonPressed(_ sender: Any) {
-        let title = titleTextField.text
-        print("my title is: \(title!)")
         
-        let notes = bodyTextView.text
-        print("my body is: \(notes!)")
         
-        UserDefaults.standard.set(title, forKey: "Title")
-        UserDefaults.standard.set(notes, forKey: "Body")
+        let alert = UIAlertController(title: "Welcome", message: "Would you like to save your note?", preferredStyle: .alert)
+        alert.addAction(UIAlertAction(title: "No", style: .cancel, handler: nil))
+    
+        alert.addAction(UIAlertAction(title: "Yes", style: .default, handler: { (UIAlertAction) in
+            if let title = self.titleTextField.text, let notes = self.bodyTextView.text {
+                print("my title is: \(title)")
+                print("my body is: \(notes)")
+                
+                UserDefaults.standard.set(title, forKey: "Title")
+                UserDefaults.standard.set(notes, forKey: "Body")
```

Comment:
Good job keeping the keys the same as the previous version of the app. Since they are used in this function and in the `handleLoadButtonPressed(...)` function, can you extract them out into constants? That is a best practice. Since if, for example, the codebase was larger and you want to change the key from like "Title" to "title_key", then you'd have to update all the usages and might miss one. Extracting it into a constant solves for that.